201-servicebus-private-endpoint/README.md

```diff
@@ -0,0 +1,96 @@
+- The virtual machine is created with a managed identity which is assigned the contributor role at the resource group scope level
+- A Public IP for the Linux virtual machine
+- The NIC used by the Linux virtual machine that makes use of the Public IP
+- A Linux virtual machine used for testing the connectivity to the storage account via a private endpoint
+- A Log Analytics workspace used to monitor the health status of the Linux virtual machine
+- A Blob Storage Account used to store data on the Azure Data Lake File System
+- A Service Bus namespace
+- A Private DNS Zone for Blob private endpoints
+- A Private DNS Zone for Service Bus private endpoints
+- A Private Endpoint to let the virtual machine access the Blob Storage Account via a private address
+- A Private Endpoint to let the virtual machine access the messaging entities in the Service Bus namespace via a private address
+
+The ARM template uses the [Azure Custom Script Extension](https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/custom-script-linux) to download and run the following Bash script on the virtual machine. The script performs the following steps:
+
+- Validates the parameters received by the Custom Script extension
+- Updates the system and upgrade packages
+- Runs the nslookup command against the public URL of the Storage Account to verify that this gets resolved to a private address
+- Runs the nslookup command against the public URL of the Service Bus namespace to verify that this gets resolved to a private address
+
+**servicebus_nslookup.sh**
+
+```bash
+#!/bin/bash
+
+# Variables
+serviceBusNamespaceEndpoint=$1
+blobServicePrimaryEndpoint=$2
+
+# Parameters validation
+if [[ -z $serviceBusNamespaceEndpoint ]]; then
+    echo "serviceBusNamespaceEndpoint parameter cannot be null or empty"
+    exit 1
+fi
+
+if [[ -z $blobServicePrimaryEndpoint ]]; then
+    echo "blobServicePrimaryEndpoint parameter cannot be null or empty"
+    exit 1
+fi
+
+# Eliminate debconf warnings
+echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+# Update the system
+sudo apt-get update -y
+
+# Upgrade packages
+sudo apt-get upgrade -y
+
+# Run nslookup to verify that public hostname of the Service Bus namespace
+# is properly mapped to the private address of the provate endpoint
+nslookup $serviceBusNamespaceEndpoint
+
+# Run nslookup to verify that public hostname of the Blob storage account 
+# is properly mapped to the private address of the provate endpoint
+nslookup $blobServicePrimaryEndpoint
+```
+
+## Deployment ##
+
+You can use the template.json ARM template and parameters.json file included in this repository to deploy the sample. Make sure to edit the parameters.json file to customize the installation. You can also use the deploy.sh Bash script under the scripts folder to deploy the ARM template. The following figure shows the resources deployed by the ARM template in the target resource group.

```

Comment:
just remove this section since it's the same for every sample in the repo (and it's not correct ;))

Reply:
I did not remove the section as the image is different :) The wording is the same, the picture of the resources in the resource group are different :)